packages/ember-metal/lib/run_loop.js

```diff
@@ -16,11 +12,7 @@ function onEnd(current, next) {
   run.currentRunLoop = next;
 }
 
-const backburner = new Backburner(['sync', 'actions', 'destroy'], {
-  sync: {
```

Comment:
this functionality in backburner was only for the sync queue

Reply:
ya, we can remove it from BB and then cut a major

Reply:
Ya, we will have to wait for a short while (because the legacy addon still needs to enable this functionality through Ember 3.4), but totally agree that it will be awesome to get rid of 😆